contracts/adapters/Withdraw.sol

```diff
@@ -54,10 +63,14 @@ contract WithdrawContract is DaoConstants, MemberGuard {
         );
 
         BankExtension bank = BankExtension(dao.getExtensionAddress(BANK));
-
+        //TODO do we need to check if the token is supported by the DAO?
+        //require(bank.isTokenAllowed(token), "token not allowed");
```

Comment:
@adridadou we may need to enable this check, but first want to verify that with you.